form/form_test.go

```diff
@@ -414,3 +454,13 @@ func TestValues(t *testing.T) {
 
 	assert.Nil(t, values.Get("boguskey"))
 }
+
+//
+// Private functions
+//
+
+// Trivial helper to get is a string pointer from a string (because you're not
+// allowed to take the address of a literal directly).
+func stringPtr(s string) *string {
```

Comment:
Why no use `stripe.String()` instead? Is it because form is "deeper" and should rely on a Stripe method?

Reply:
Yeah, my justification is that essentially the `form` package should be considered kind of a utility that's at a lower level than anything Stripe-related. I wouldn't mind using `stripe.String` that much, but I figure that this implementation keeps boundaries a little cleaner, and the function is so trivial that it's never going to change or anything.